apps/web/pages/apps/index.tsx

```diff
@@ -3,24 +3,20 @@ import { InferGetStaticPropsType } from "next";
 import { getAppRegistry } from "@calcom/app-store/_appRegistry";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import prisma from "@calcom/prisma";
-import Shell from "@calcom/ui/Shell";
+import { AppCategories } from "@calcom/prisma/client";
+import AppsLayout from "@calcom/ui/v2/core/layouts/AppsLayout";
 
-import AppsShell from "@components/AppsShell";
 import AllApps from "@components/apps/AllApps";
 import AppStoreCategories from "@components/apps/Categories";
-import TrendingAppsSlider from "@components/apps/TrendingAppsSlider";
 
 export default function Apps({ appStore, categories }: InferGetStaticPropsType<typeof getStaticProps>) {
   const { t } = useLocale();
 
   return (
-    <Shell heading={t("app_store")} subtitle={t("app_store_description")} large isPublic>
-      <AppsShell>
-        <AppStoreCategories categories={categories} />
-        <TrendingAppsSlider items={appStore} />
-        <AllApps apps={appStore} />
-      </AppsShell>
-    </Shell>
+    <AppsLayout heading={t("app_store")} subtitle={t("app_store_description")}>
+      <AppStoreCategories categories={categories} />
+      <AllApps apps={appStore} />
+    </AppsLayout>
```

Comment:
is there a point of updating the `v1` layout here?

Reply:
Weird, I reverted that already 🤔  https://github.com/calcom/cal.com/blob/feat/v2-app-store/apps/web/pages/apps/index.tsx